spec/jobs/segment_track_job_spec.rb

```diff
@@ -19,36 +21,36 @@
       expect(SEGMENT_CLIENT).to receive(:track)
         .with(
           user_id: membership_id,
-          event: event,
+          event:,
           properties: {
             method: 1,
             hosting_type: 'self',
-            version: Utils::VersionService.new.version.version.number
-          }
+            version: Utils::VersionService.new.version.version.number,
+          },
         )
 
-      subject.perform_now(membership_id: membership_id, event: event, properties: properties)
+      subject.perform_now(membership_id:, event:, properties:)
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)